### PR TITLE
Update the Add Controller doc and fix some line numbers

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ryanbrandenburg  I'm right in the middle of updating this. Maybe it's better if I take you changes in.

File: aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ﻿{
   "dependencies": {
     "Microsoft.NETCore.App": {
-      "version": "1.0.0-rc3-004443-00",
+      "version": "1.0.0",
       "type": "platform"
     },
-    "Microsoft.AspNetCore.Diagnostics": "1.0.0-rc3-21343",
-    "Microsoft.AspNetCore.Mvc": "1.0.0-rc3-21343",
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0-*",
```
Comment (Rick-Anderson) [inline aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json]: The last version I checked in has "Microsoft.AspNetCore.Diagnostics": "1.0.0",

File: aspnet/tutorials/first-web-api.rst [hunk 2]
```diff
@@ -115,8 +115,8 @@ In the ``ConfigureServices`` method, add the highlighted code:
 
 .. literalinclude:: first-web-api/sample/src/TodoApi/Startup.cs
   :language: c#
-  :lines: 25-31
-  :emphasize-lines: 6
+  :lines: 14-23
```
Comment (Rick-Anderson) [inline aspnet/tutorials/first-web-api.rst]: /tutorials/first-web-api.rst should be in a separate PR

File: aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json [hunk 3]
```diff
@@ -31,11 +31,11 @@
       ]
     },
     "Microsoft.AspNetCore.Razor.Tools": {
-      "version": "1.0.0-preview2-21343",
+      "version": "1.0.0-*",
```
Comment (ajaybhargavb) [inline aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json]: These should be `preview2-final`

Comment (Rick-Anderson) [inline aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json]: PR https://github.com/aspnet/Docs/pull/1457 will be built with the final RTM bits and should have all the correct version numbers. As mentioned previously, any good changes here I'll manually copy to my PR

**Comment by Rick-Anderson** [comment]
I took all your changes. I'm getting rid of all line numbers - so updated it my markup to show routes to the following: Thanks for these nice improvements.

.. literalinclude:: start-mvc/sample2/src/MvcMovie/Startup.cs
  :language: c#
  :start-after: app.UseIdentity();
  :end-before:  SeedData.Initialize(app.ApplicationServices);
  :dedent: 12
  :emphasize-lines: 5
